code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5158,7 +5158,7 @@ var/procizine_tolerance = 0
 				if(has_mouse_bulked == 0)
 					if(prob(95))
 						has_mouse_bulked = 1
-						if(prob(90))
+						if(prob(95))
```

Comment:
I liked the 90% better. Oh well.

Reply:
I don't want it to be way too common. A 1 out of 20 chance is fine enough.

Reply:
I do think the way this snippet of probabilities is pretty ugly, though.

If anyone has a suggestion for a better way to write this, please help.